sympy/assumptions/ask.py

```diff
@@ -1382,6 +1386,12 @@ def ask(proposition, assumptions=True, context=global_assumptions):
 
         It is however a work in progress.
 
+    See Also
+    ========
+
+    sympy.assumptions.refine.refine : Simplification using assumptions.
```

Comment:
Can this just be `refine :`?

Reply:
That raised sphinx failure. It said that multiple target for cross referencing was found.

Reply:
Is there any way to overcome this? See Also section does not take `:func:` syntax so I can't figure out how to enhance this.